spring-data-jdbc/src/test/java/org/springframework/data/jdbc/core/convert/SqlGeneratorUnitTests.java

```diff
@@ -392,11 +394,11 @@ public void updateWithVersion() {
 	@Test // DATAJDBC-264
 	public void getInsertForEmptyColumnList() {
 
-		SqlGenerator sqlGenerator = createSqlGenerator(IdOnlyEntity.class);
+		SqlGenerator sqlGenerator = createSqlGenerator(IdOnlyEntity.class, PostgresDialect.INSTANCE);
 
 		String insert = sqlGenerator.getInsert(emptySet());
 
-		assertThat(insert).endsWith("()");
+		assertThat(insert).endsWith(PostgresDialect.INSTANCE.getSqlInsertWithDefaultValues().getDefaultInsertPart());
```

Comment:
Again, just hard code the expected String. That is much easier to understand.

Also: we should have a second test testing for some other variant of the behaviour using a diiferent dialect.

Reply:
Resolved [here](https://github.com/spring-projects/spring-data-jdbc/pull/1047/commits/1841135d9b2f60424d0c60460569ac8d99d39ecd)